internal/tool/os_util.go

```diff
@@ -0,0 +1,71 @@
+// Licensed to SkyAPM org under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. SkyAPM org licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package tool
+
+import (
+	"net"
+	"os"
+	"runtime"
+	"strconv"
+)
+
+func ProcessNo() string {
```

Comment:
Why I can't find where using this?

Reply:
[here](https://github.com/arugal/go2sky/blob/a9a8a9cbd19ee3b100db316e1cc432aac969290b/reporter/grpc.go#L268-L307)